ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js

```diff
@@ -100,6 +102,19 @@ describe("Form with Submit Button", () => {
         });
     })
 
+
+    if (cy.af.isLatestAddon()) {
+        it("Custom Submit Action Localisation Test", () => {
+            cy.previewForm(customSubmitLocalisationPagePath);
+            cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
```

Comment:
where is the x used? if you don't need a variable, you can use empty brackets () instead